spyder/utils/syntaxhighlighters.py

```diff
@@ -332,6 +332,7 @@ def make_python_patterns(additional_keywords=[], additional_builtins=[]):
     builtin = r"([^.'\"\\#]\b|^)" + any("builtin", builtinlist) + r"\b"
     comment = any("comment", [r"#[^\n]*"])
     instance = any("instance", [r"\bself\b",
+				r"\bcls\b",

```

Comment:
Fix your alignment; should be one space to the right.

Reply:
I can't find any test case for the function "make_python_patterns".
Should I add a new test in the "test_syntaxhighlighters.py"?

https://github.com/spyder-ide/spyder/blob/master/spyder/utils/tests/test_syntaxhighlighters.py

Reply:
Sure, if you don't mind.